src/logger.h

```diff
@@ -1,4 +1,28 @@
 #pragma once
+
+#include <fstream>
+
 #include <boost/log/trivial.hpp>
+#include <boost/log/utility/setup/from_stream.hpp>
+#include <boost/log/core/core.hpp>
+#include <boost/log/sources/logger.hpp>
+
+namespace logging = boost::log;
+namespace src = boost::log::sources;
+using LoggerType = src::logger_mt;
+
+#define CREATE_LOCAL_LOGGER(logger) \
+  src::logger_mt logger
+
+#define CREATE_GLOBAL_LOGGER(logger) \
+namespace { \
+  GET_LOCAL_LOGGER(); \
```

Comment:
@AGaliuzov, I'm not sure how boost works. I've just read that boost is not hierarchical and I see no names passed through ctor  so I'm not sure how names are assigned. We'll see...